config.py

```diff
@@ -25,6 +25,10 @@ class CannotLoadConfiguration(Exception):
     pass
 
 
+class CannotSendEmail(Exception):
+    pass
```

Comment:
New exception type, raised when an `Emailer` instance's `send()` method gets an exception when it tries calling `_send_email()`.